.github/workflows/dart_ci.yml

```
@@ -38,6 +38,26 @@ jobs:
         run: dart test
         if: always() && steps.install.outcome == 'success'
 
+      - id: install-tools
```

Looks like the rest of these are not special cased
```suggestion
      - id: Install tools
```

---

I believe this id is used in the steps.install-tools.outcome evaluations, so this wouldn't work.

However, it seems like these should all be using a `needs` clause instead. But I didn't want to make that change and be inconsistent.